### ISSUE TITLE
Linkhut: LINK_ASIO_DRIVER_NAME shouldn't be hardcoded to "ASIO4ALL v2"

### ISSUE THREAD

**Issue by JoergAtGithub** [Ableton/link#126]
ASIO4ALL is just an uneccessary translation layer. It's in general better to use the ASIO audio driver provided by the device manufacturer. This shouldn't require to recompile Linkhut.
Expected behavior:

- If only one ASIO driver is installed, Linkhut should use this
- If multiple ASIO drivers are installed Linkhut should ask at startup, which of them should be used e.g.:
  `[1] ASIO4ALL v2`
  `[2] Native Instruments Traktor Z2`
  `Enter number of ASIO driver to use and press Enter`

**Comment by cdi-ableton** [comment]
I agree that an application should use the audio interface's dedicated driver. However, our goal for Link's example application was to showcase the Link integration with as little clutter around it as possible and to avoid additional code for different platforms.
